components/no_results_indicator/no_results_indicator.tsx

```diff
@@ -58,30 +65,63 @@ const subtitleMap: {[key in NoResultsVariant]: MessageDescriptor } = {
     },
 };
 
-const NoResultsIndicator = (props: Props) => {
-    return (
+import './no_results_indicator.scss';
+
+const NoResultsIndicator = ({
+    expanded,
+    style,
+    variant,
+    iconGraphic = variant ? (
```

Comment:
Can we inline these instead of writing the defaults in this way?

Reply:
The multiple props/attrs on the `FormattedMessage`s force multiline per eslint, so I wrapped them for consistency. And I opted for defaults vs intl.formatMessage() to simplify the func/render body, but if there's another way to format them, I don't have much preference either way.